code/utility/logger.cpp

```diff
@@ -11,6 +11,7 @@ void Logger::setStream(QTextEdit* log_stream) {
 
 //TODO: Overload ostream operator
 bool Logger::log (std::string message, LogType type) {
+    if (type == LogType::DEBUG && !Logger::m_debug) return false;

```

Comment:
I used #ifdef QT_DEBUG ... at the top of this file, which is defined regardless of whether I run normally or in debug mode (in Qt creator). 

I'm told that QT_NO_DEBUG is defined instead for release builds, however. 

Let me know if this is what you want